src/ui/src/js/controllers/admin_system.js

```diff
@@ -272,6 +273,11 @@ export default function adminSystemController(
     $scope.runnerResponse = response;
     $scope.runners = response.data;
 
+    // This is kind of messy, but oh well
+    if (responseState($scope.response) === 'empty') {
+      $scope.response = response;
+    }
+
```

Comment:
So what is actually going on here? If nothing is returned, then set the `$scope.response` to an empty list?

Reply:
Yeah, I'm not super happy about this, but it's at least an improvement until we get the new system admin page.

So you know the big yellow display that says "Whoops, couldn't find that" when you go to the system admin page when there aren't any systems? That's based off of the return code of `$scope.response`. So even if there are unassociated runners we were still showing that big banner.

This change just hides the banner for that specific case.

The root cause of this is that we're combining two "queries" (systems and runners) into one logical grouping. Our frontend isn't that flexible so I did this 😄